packages/lib/src/spatial-navigation/components/ScrollView.tsx

```diff
@@ -59,9 +129,27 @@ export const SpatialNavigationScrollView = ({
         showsHorizontalScrollIndicator={false}
         showsVerticalScrollIndicator={false}
         scrollEnabled={false}
+        onScroll={(event) => {
+          scrollY.current = event.nativeEvent.contentOffset.y;
+        }}
+        scrollEventThrottle={16}
       >
         {children}
       </ScrollView>
+      {deviceType === 'remotePointer' ? (
+        <>
+          <TopArrowContainer style={topArrowContainerStyle} {...webProps(onMouseEnterTop)}>
```

Comment:
Ideally we'd like to memoize this as well, if possible

```suggestion
          <TopArrowContainer style={topArrowContainerStyle} {...onMouseEnterTopWebProps}>
```

Reply:
Fixed